apps/cloud_composer/include/pcl/apps/cloud_composer/tools/impl/supervoxels.hpp

```diff
@@ -95,11 +96,12 @@ pcl::cloud_composer::SupervoxelsTool::performTemplatedAction (const QList <const
     std::map <std::uint32_t, typename pcl::Supervoxel<PointT>::Ptr > refined_supervoxel_clusters;
     super.refineSupervoxels (3, refined_supervoxel_clusters);
   
-    typename pcl::PointCloud<PointXYZRGBA>::Ptr color_segments;
-    color_segments= super.getColoredVoxelCloud ();
-    
-    CloudItem*  cloud_item_out = CloudItem::createCloudItemFromTemplate<PointXYZRGBA>(input_item->text(),color_segments);
- 
+    typename pcl::PointCloud<PointXYZL>::Ptr label_segments = super.getLabeledVoxelCloud ();
```

Comment:
Did you miss this one or is `typename` actually required?

Reply:
I switched off my brain and applied your comment to a single line above the comment :laughing:

Reply:
I was so proud for even using the new functionality of comments spanning multiple lines. :x

Reply:
And now I see that yours was a multi-line comment even :see_no_evil: